hack/deploy_dev_llm.sh

```diff
@@ -9,6 +9,16 @@ if ! command -v ko &>/dev/null; then
     go install github.com/google/ko@latest
 fi
 
+echo "Creating Istio Gateway ..."
+kubectl create ns kserve || true
+# Replace gatewayclass name
+kubectl apply -f - <<EOF
+$(sed 's/envoy/istio/g' config/overlays/test/gateway/ingress_gateway.yaml)
+EOF
+sleep 10
```

Comment:
instead sleeping, isn't better to check for the resource and keep trying for a few seconds if not available.
There is a func in some of our scripts that does that, I don't recall the name of the function, something related to `wait_for_resource`.